osm2streets/src/lanes/mod.rs

```diff
@@ -136,6 +139,16 @@ impl LaneType {
         )
     }
 
+    pub fn to_traffic_mode(&self) -> Option<TrafficMode> {
+        use LaneType::*;
+        match self {
+            Bus | SharedLeftTurn | Driving => Some(TrafficMode::Motor),
+            SharedUse | Biking => Some(TrafficMode::Bike),
```

Comment:
Just nitpicking on the meaning here - this is the "fastest" mode that primarily uses this lane type. Of course bikes also use the above lanes

Reply:
Yes, the usecases in this PR care about the "biggest" kind of traffic, which indicates the kind of markings that will be present. A shared foot/cycle path will have cycle markings but a shared bus/cycle lane will have road markings.

There's probably a better method name for it.

Reply:
I have introduced the term "Traffic Class" for this, and added `Rail` as a variant. I realised my goal here is to distinguish between the distinct kinds of infrastructure: paths, tracks, roads and rail, as I've talked about in #138, looking at roads before paths.